core/sdk/src/api/firebolt/fb_metrics.rs

```diff
@@ -827,6 +625,18 @@ impl MetricsContext {
             authenticated: None,
         }
     }
+
+    pub fn get_fw_version(&self) -> String {
```

Comment:
Seems RDK Release specific perhaps extension might be a better fit

Reply:
do u mean the name of the function is too RDK release specific, or that this function should be in the extension?

Reply:
yeah terms VBN and PROD are naming conventions used by RDK Release. Feel they would change in the future impacting our Ripple core. Having them in extension before they are exported seems a better strategy